OWML.ModLoader/ModLoader.cs

```diff
@@ -25,17 +27,18 @@ public static void LoadMods()
                 // Everything is wrong and can't write to console...
                 return;
             }
-            var logger = new ModLogger(owmlConfig, owmlManifest);
+            var logger = new ModLogger(owmlConfig, owmlManifest, startTime);
```

Comment:
looks like you could create the filename first and pass that in, instead of the time